src/it/scala/com/ing/wbaa/gargoyle/sts/StsServiceItTest.scala

```diff
@@ -55,17 +61,24 @@ class StsServiceItTest extends AsyncWordSpec with DiagrammedAssertions
 
       override protected[this] def stsSettings: GargoyleStsSettings = GargoyleStsSettings(testSystem)
 
-      override protected[this] def gargoyleNPASettings: GargoyleNPASettings = GargoyleNPASettings(testSystem)
+      override protected[this] def gargoyleMariaDBSettings: GargoyleMariaDBSettings = new GargoyleMariaDBSettings(testSystem.settings.config)
+
+      override protected[this] def insertToken(awsSessionToken: AwsSessionToken, username: UserName, expirationDate: AwsSessionTokenExpiration, assumedGroup: Option[UserAssumedGroup]): Future[Boolean] =
+        Future.successful(true)
+
+      override protected[this] def getToken(awsSessionToken: AwsSessionToken): Future[Option[(UserName, AwsSessionTokenExpiration, UserAssumedGroup)]] =
+        Future.successful(Some((UserName("u"), AwsSessionTokenExpiration(Instant.now()), UserAssumedGroup("group"))))
 
       override def generateAwsCredential: AwsCredential = AwsCredential(
-        AwsAccessKey("accesskey" + Random.alphanumeric.take(32).mkString),
-        AwsSecretKey("secretkey" + Random.alphanumeric.take(32).mkString)
+      AwsAccessKey("accesskey" + Random.alphanumeric.take(32).mkString),
+      AwsSecretKey("secretkey" + Random.alphanumeric.take(32).mkString)
       )
 
       override def generateAwsSession(duration: Option[Duration]): AwsSession = AwsSession(
-        AwsSessionToken("sessiontoken" + Random.alphanumeric.take(32).mkString),
-        AwsSessionTokenExpiration(Instant.now())
```

Comment:
not sure if that's the issue here, but be sure to run `sbt clean assembly` (or any other sbt clean command) before you push. Sbt performs the sourcecode formatting to make sure we don't have these types of commits with spacing changes.